tasks/upgrade.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Is there actually any overlap between these two tasks? Would it make more sense to have a separate upgrade role?